src/commands/link.ts

```diff
@@ -0,0 +1,35 @@
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder } from "@discordjs/builders";
+
+const LINKS: { [k: string]: string } = {
+  docs: "https://docs.codewars.com/",
+  honor: "https://docs.codewars.com/gamification/honor",
+  kata: "https://www.codewars.com/kata",
+  katas: "https://www.codewars.com/kata",
+  authoring: "https://docs.codewars.com/authoring",
+  rank: "https://docs.codewars.com/curation/kata",
+  ranking: "https://docs.codewars.com/curation/kata",
+  troubleshooting: "https://docs.codewars.com/training/troubleshooting",
+  debug: "https://docs.codewars.com/training/troubleshooting",
+  github: "https://github.com/codewars/",
+  clans: "https://docs.codewars.com/community/following/#clans",
+  vim: "https://docs.codewars.com/references/kata-trainer/#editors",
+};
+
+// link
+export const data = new SlashCommandBuilder()
+  .setName("link")
+  .setDescription("Link to a given topic")
+  .addStringOption((option) =>
+    option
+      .setName("topic")
+      .setDescription("The topic to link to")
+      .setRequired(true)
+      .addChoices(Object.keys(LINKS).map((k) => [k, k]))
+  )
+  .toJSON();
+
+export const call = async (interaction: CommandInteraction) => {
+  const topic = interaction.options.getString("topic", true);
+  await interaction.reply(`See <${LINKS[topic]}>`);
```

Comment:
What do you think of hiding the url? [`hyperlink(description, url)`](https://discordjs.guide/popular-topics/builders.html#links) can be used. `LINKS` can map a short trigger keywords to url and a descriptive text. And that descriptive text can be used as a description for choices too, so it's clear what each means.

```
/link topic:katas

See [kata search page](https://www.codewars.com/kata)
```

Reply:
Thanks, I'll look into it.

Another thing I noticed is that slash commands registered by our bot cannot be used as a reply to existing comments, so I'll probably have to add an extra parameter to have the bot mention a specific user when posting the link.

Reply:
Is that a known limitation? Or is our bot missing some required scope?

Reply:
Seems to me that it's a limitation of bot-registered slash commands. Anyway, I've added an optional parameter for directing a specific user to the selected link through mentions, let me know what you think.